README.md

```diff
@@ -19,32 +21,41 @@ composer require fortuneglobe/types
 
 ### Available base types
 
-* StringType
-* FloatType
-* IntType
-* UuidType (extends StringType)
+* AbstractStringType
+* AbstractFloatType
+* AbstractIntType
+* AbstractUuidType (extends AbstractStringType)
```

Comment:
It's `AbstractUuid4Type`.

Reply:
Fixed in 1fcd1e80b61d3f6650e3ed11cc2b2d3b3deb4f0b